tests/EdFi.Ods.Api.IntegrationTests/StudentUSIToEducationOrganizationIdAuthViewTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using NUnit.Framework;
+using System;
+
+namespace EdFi.Ods.Api.IntegrationTests
+{
+    [TestFixture]
+    public class StudentUSIToEducationOrganizationIdAuthViewTests : DatabaseTestFixtureBase
+    {
+        [Test]
+        public void When_calling_studentUSI_to_educationOrganizationId_authView_should_not_get_duplicaterecords()
+        {
+             AuthorizationViewHelper.ShouldNotContainDuplicate(Connection, "StudentUSIToEducationOrganizationId");
+        }
+
+        [Test]
+        public void When_enroll_student_in_school_should_get_access_from_school()
+        {
+            var studentUniqueId = Guid.NewGuid().ToString("N");
+
+            Builder
+                .AddSchool(9701).AddStudent(studentUniqueId)
+                .Execute();
+
+            var studentUSI = Builder
+                 .GetStudentUSI(studentUniqueId)
+                 .ExecuteScalar();
+
+            Builder
+                .AddStudentSchoolAssociation(9701, studentUSI, Builder.TestGradeLevelDescriptorId)
+                .Execute();
+
+            AuthorizationViewHelper.ShouldContainStudentInSchoolOrDistrict(Connection, "StudentUSIToEducationOrganizationId", (9701, studentUSI));            
+        }
+
+        [Test]
+        public void When_enroll_student_in_school_should_not_access_from_other_school()
+        {
+            var studentUniqueId = Guid.NewGuid().ToString("N");
+
+            Builder
+                .AddSchool(9702).AddStudent(studentUniqueId)
+                .Execute();
+
+            var studentUSI = Builder
+                 .GetStudentUSI(studentUniqueId)
+                 .ExecuteScalar();
+
+            Builder
+                .AddStudentSchoolAssociation(9702, studentUSI, Builder.TestGradeLevelDescriptorId)
+                .Execute();
+
+            AuthorizationViewHelper.ShouldNotContainStudentInOtherSchoolOrDistrict(Connection, "StudentUSIToEducationOrganizationId", (9702, studentUSI));
```

Comment:
since `ShouldNotContainStudentInOtherSchoolOrDistrict` is the same functionality as ShouldContain... I don't think this is testing what we'd expect

Reply:
fixed it .Thanks